src/pudl/transform/eia923.py

```diff
@@ -587,9 +587,9 @@ def gen_fuel_nuclear(gen_fuel_nuke: pd.DataFrame) -> pd.DataFrame:
     outs={
         "clean_generation_fuel_eia923": AssetOut(),
         "clean_generation_fuel_nuclear_eia923": AssetOut(),
-    }
+    },
 )
-def generation_fuel_eia923(raw_generation_fuel_eia923: pd.DataFrame):
+def clean_generation_fuel_eia923(raw_generation_fuel_eia923: pd.DataFrame):
```

Comment:
Hmm I'm not sure if that's a bug. It throws reasonable errors when you have duplicate asset names but it does seem strange it throws an error when a multi asset function name conflicts with an asset. I have a feeling an error will not be thrown if the function name conflicts with an asset name but the multi_asset name parameter is a unique name.

Reply:
I guess I thought that the function-name wouldn't show up anywhere in the dagster abstractions, because the assets / outputs were being set separately / explicitly rather than being derived automatically from the function name.

Reply:
Still weirdly confused by this.

Reply:
I also wouldn't expect the function name of a multi_asset to produce a name collision error. Maybe we should open an issue?